card/mod/05_standard/set/all/attach.rb

```diff
@@ -120,23 +120,21 @@ module Paperclip::Interpolations
   
   extend Card::Format::Location
 
-  def local at, style_name
+  def system_path at, style
+    card = at.instance
     if mod = at.instance.attach_mod
       # generalize this to work with any mod (needs design)
-      "#{Cardio.gem_root}/mod/#{mod}/file"
+      codecard = card.left if card.cardname.junction?
+      "#{ Cardio.gem_root}/mod/#{mod}/file/#{codecard.codename}/#{size at, style}#{card.type_code}"
     else
-      Card.paths['files'].existent.first
+      "#{ Card.paths['files'].existent.first }/#{card.id}/#{size at, style}#{action_id at, style}"
     end
```

Comment:
this handling is significantly different.  the idea is that if you want to have file/image content in a mod, it's not really safe to base the handling on ids -- those could be different in every deck.

We got away with this before because we were only using it in new decks, where we could control the ids.